### PR TITLE
Init gyro_tokens

### PR DESCRIPTION
**Pull request by viniabussafi**
This PR creates a proxy for the price of the tokens that were submitted on https://github.com/duneanalytics/spellbook/pull/5619#event-12155682375, that were recently launched and will be featured in gyroscope pools

@mendesfabio

### CODE REVIEW

File: models/gyroscope/gyro_tokens.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{ config(
+    schema = 'gyroscope',
+    alias = 'gyro_tokens'
```
Comment (Hosuke) [inline models/gyroscope/gyro_tokens.sql:3]: If the schema is `gyroscope`,
then the model name would be `gyroscope_gyro_tokens`.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/gyroscope/gyro_tokens.sql (hunk 1 above)
Comment (viniabussafi) [reply models/gyroscope/gyro_tokens.sql:3]: yup, sorry for the oversight

**Review by jeff-dude — APPROVED** [review]
if i understand correctly, when these tokens are traded enough on DEXs, eventually the price derived in `dex.prices` will suffice and you can essentially revert this PR?

**Comment by viniabussafi** [comment]
Exactly, @jeff-dude! This is just a way to get things right as soon as pools are launched

**Pull request merged**